package.json

```diff
@@ -23,10 +23,10 @@
   "author": "GitHub",
   "license": "MIT",
   "dependencies": {
-    "@actions/core": "^1.2.0",
-    "@actions/exec": "^1.0.1",
-    "@actions/io": "^1.0.1",
-    "@actions/cache": "^1.0.1"
+    "@actions/core": "^1.2.4",
+    "@actions/exec": "^1.0.4",
+    "@actions/io": "^1.0.2",
+    "@actions/cache": "^1.0.2"
```

Comment:
Can we only update `@actions/cache`? We intentionally do not update all NPM packages to the latest versions unless required.

`npm update @actions/cache` and then `npm install` and `npm run release` should do the trick

Reply:
I changed it.